frontend/src/app/shared-components/shared-components.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule } from '@angular/core';
+import { ToolbarComponent } from './toolbar/toolbar.component';
+import { MatToolbarModule } from '@angular/material/toolbar';
+import { MatIconModule } from '@angular/material/icon';
+import { SharedComponentsRoutingModule } from './shared-components-routing.module';
+import { CommonModule } from '@angular/common';
+
+@NgModule({
+  declarations: [ToolbarComponent],
+  exports: [ToolbarComponent],
+  imports: [MatToolbarModule, MatIconModule, SharedComponentsRoutingModule, CommonModule],
```

Comment:
**thought:** worth reading through what the CommonModule does to understand modules better

Reply:
It gives me access to all the common directives and pipes, such as *ngIf and *ngFor . The toolbar component requires the *ngIf directive, hence the import.